### DISCUSSION TITLE
Any plans to support GitLab CI/CD pipelines?

### DISCUSSION THREAD

**Discussion by hbendix** [mikaelvesavuori/figmagic#172] [General]
Figmagic looks to support our workflow down to a T, however, I noticed your continuous design package is yet to mention GitLab. Is this planned for the future? I was unable to find a roadmap of sorts.

Keep up the good work! 

Harry

**Comment by mikaelvesavuori** [comment]
Hey @hbendix!

This will sound really dumb but I never got informed something happened in the GitHub Discussions... Sorry about that!

There has been no plan to support GitLab previously, but I would be happy to support if you pitched in!
EDIT: See below

**Reply by mikaelvesavuori** [reply]
Oh, and sorry for missing the obvious: given that this pertains to the [Figma plugin](https://github.com/mikaelvesavuori/figma-plugin-continuous-design) I will close this discussion. The only relation is that I give examples (and provide UI) to call a workflow in some different providers (GH, Azure...) where Figmagic happens to be used in the demo scripts.

**Reply by mikaelvesavuori** [reply]
Hey again @hbendix!

Support for GitLab is now out at https://www.figma.com/community/plugin/977948326423807703 (also see, if needed, https://github.com/mikaelvesavuori/figma-plugin-continuous-design).
